### ISSUE TITLE
libcef crash on Ubuntu 13.10 with segfault

### ISSUE THREAD

**Issue by makson96** [desura/Desurium#684]
http://youtu.be/uzflheiVHGg
After starting Desurium on Ubuntu 13.10 in Virtualbox it crashes with fallowing output:

```
08:49:41: Deleted stale lock file '/home/makson/.desura/cache/applock'.
/usr/bin/desurium: line 21: 2287 Naruszenie ochrony pamięci (core dumped) $DIR/lib/desura $@
```

"Naruszenie ochrony pamięci" can be translated as a segfault. I am not sure if bug is connected to Ubuntu 13.10 or rather using Virtualbox.
If I remove libcef-desura, than Desurium works fine, but of course it does not display any web content.

**Comment by karolherbst** [comment]
without a stacktrace or crashdump this information is kind of useless. Is it reproducable  and is there any information you could give us?

**Comment by makson96** [comment]
Yes it is reproducible. Actually it crash every single time I am trying to start it on Ubuntu 13.10. When I will have more time I will try to provide more info through stacktrace or crashdump.
But it should be very easy to reproduce by anyone - just start Desurium on Ubuntu 13.10.

**Comment by makson96** [comment]
I hope this is helpful. Valgrind report:
http://pastebin.com/kXFRFmay

**Comment by lodle** [comment]
did you compile it from scratch on 13.10?

**Comment by makson96** [comment]
I used Launchpad servers to build it and downloaded binaries directly from repository:
https://launchpad.net/~makson96/+archive/desurium

**Comment by adrianheine** [comment]
I also get this error with @makson96’s binaries on debian sid.
